peewee_async.py

```diff
@@ -697,13 +297,6 @@ def execute_sql(self, *args, **kwargs):
                         (str(args), str(kwargs)))
         return super().execute_sql(*args, **kwargs)
 
-    async def fetch_results(self, query, cursor):
-        if isinstance(query, peewee.BaseModelSelect):
-            return await AsyncQueryWrapper.make_for_all_rows(cursor, query)
-        if isinstance(query, peewee.RawQuery):
-            return await AsyncQueryWrapper.make_for_all_rows(cursor, query)
-        raise Exception("Unknown type of query")
```

Comment:
I remembered. Since ModelRaw.aio_execute() will not work database.aio_execute will work otherwise for such queries. database.aio_execute is a replace for manager.execute. Let's revert these lines until we do #236 and #237

Reply:
@kalombos I suggest to move forward, not backward -- if we already extracting compat layer, let's use it 100%. Check the next commit, compat patching is applied in `Database.aio_execute()` and removed from a bunch of other places. Now it's even more clear and straightforward. The compound select test is also updated to use `Database.aio_execute()`.

Reply:
I agree it looks good and aio_execute works now